libs/input-mapping/src/Table/Options/RewrittenInputTableOptions.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\InputMapping\Table\Options;
+
+use Keboola\InputMapping\Exception\TableNotFoundException;
+use Keboola\InputMapping\State\InputTableStateList;
+
+class RewrittenInputTableOptions extends InputTableOptions
+{
+    private array $tableInfo;
+
+    public function __construct(array $definition, string $source, int $sourceBranchId, array $tableInfo)
```

Comment:
Ten `source` je tady k ničemu, když stejnou informaci dostaneš i v tom tableInfo ne?

A `sourceBranchId`, protože detail tabulky obsahuje informace o bucketu, včetně branche.

Reply:
nn, source se přepisuje to je kruciální tady https://github.com/keboola/platform-libraries/pull/129/files#diff-39b2c40c2984fdb565f040697e6ad6126b2b1eb843a9d88adaad6e8c7ccfaf54R29

Reply:
a sourceBranchId se přepisuje zase tady https://github.com/keboola/platform-libraries/pull/129/files#diff-f4ecfae2b9920a10bb469c21b0e57e72efafdffbdcf5a69fc18e01c470b120e2R31